enclave/src/channel_storage.rs

```diff
@@ -31,17 +31,41 @@ lazy_static! {
         Arc::new(SgxMutex::new(None));
 }
 
+lazy_static! {
```

Comment:
Why do we need this? Shouldn't we test with the real lazy static instead of testing the test?

Reply:
We need that because other tests already initialize and use it in the db handler thread, so if we use the same in the unit test that thread will be stuck in an infinite loop because the receiver won't match the sender

Reply:
I see. But I still think this is not an acceptable solution because we're not testing the actual implementation. We might be able to solve this with a compiler flag or something different..

Do you think we should merge this PR before the testnet? If so, I will approve under the condition that an issue tackling this problem is opened.